lib/modules/touch.js

```diff
@@ -0,0 +1,50 @@
+import { contentLoaded } from '../utils';
+
+export default (shower) => {
+    contentLoaded(() => {
```

Comment:
I am not sure `contentLoaded` is necessary here. `lib/modules/install.js` should handle the page lifecycle / events.